src/api/ApiManager.js

```diff
@@ -0,0 +1,106 @@
+import axios from "axios";
+import { BASE_URL } from "./ApiConstants";
+
+
+const getToken = () => localStorage.getItem("token");
+
+axios.interceptors.response.use(
+  (response) => response,
+  (error) => {
+    if (error.response && error.response.status === 401) {
+   
+      window.location.href = "/";
+    } else {
+      console.error("Response error:", error);
```

Comment:
There's no need for using consoles, even in api manager. You could have shown it as alert message instead.